ts/screens/services/ServicesHomeScreen.tsx

```diff
@@ -114,6 +124,7 @@ type State = {
   enableHeaderAnimation: boolean;
   isLongPressEnabled: boolean;
   enableServices: boolean;
+  toastContent?: string;
```

Comment:
I understood the goal of toastContent so I suggest these changes b14c5e4facd653b51571c23626827d1f5076d207 to make it more readable

Reply:
https://github.com/teamdigitale/io-app/pull/1352/commits/17f6c8d49eff55069550ab43938c2b79ec0a21c1 introduces most of the proposed changes. An exception is represented by the `componentDidUpdate` function (we can't avoid the previous props value is evaluated to display the error toast)